### DISCUSSION TITLE
2D periodic boundary conditions problems

### DISCUSSION THREAD

**Discussion by LorenzoFabris** [su2code/SU2#1523] [Q&A]
**2D PERIODIC BOUNDARY CONDITIONS - DIVERGENT SOLUTION**
Different test cases have been investigated in order to solve 2D-boundary conditions problems. While translational periodic boundary conditions set up perfectly worked and provided consistent results, rotational periodic boundary conditions set up lead to a divergent solution. 
The meshes have been realized with Ansys Fluent Mesher, then converted into .CGNS (adf) file format using ICEM CFD.

**To Reproduce**

The investigated scenario can be better visualized thanks in the following image:
![2021-12-22 (2)](https://user-images.githubusercontent.com/96534491/147110477-ace3e92c-225d-4973-9675-8fd6f57a8404.png)

Eventually, it was possible to observe the non-convergence of 2D rotational periodic boundary conditions solutions.
The divergent behavior above mentioned cannot be traced to deceleration effects with respect to the fluid domain, since different ratios inlet/outlet have been simulated and the solution behavior itself has never changed.  The solver crashes after 245 iterations, approximately.

Considering that 2D translational periodic boundary condition problems are converging, it is suggested that this behavior might be traced to a bug. 

In order to reproduce the simulation, here attached the .cfg file:
[Cone.txt](https://github.com/su2code/SU2/files/7763374/Cone.txt)
_(according to github, I cannot upload .CGNS file to this report)_

**Desktop:**
 - **OS:** Microsoft Windows 10 64bit
 - **C++ compiler and version:**  g++ (GCC) 1.42.0
 - **MPI implementation and version:** OpenMPI 10.1.12498.18
 - **SU2 Version:** v7.2.1 Blackbird

**Comment by pcarruscag** [comment]
Hi,
Can you post the output of the code?
Is your angle one degree? Or one radian? (the specification is in degrees)

**Comment by LorenzoFabris** [comment]
Hi Pedro, 
I ran two simulations, the first one considering one radian angle and the second one specifying my angle in degree. 
However, the solution is still divergent. 

Here following the .csv files SU2 has generated right before crashing. 

**Degree-specified Angle**
[history angle 1.csv](https://github.com/su2code/SU2/files/7838108/history.angle.1.csv)

**Radian-specified Angle**
[history.csv](https://github.com/su2code/SU2/files/7838107/history.csv)

**Comment by pcarruscag** [comment]
Hi Lorenzo,
Post the full SU2 screen output please.
You have a Navier-Stokes problem at Re 6e6, that does not sound physical.
There is also a mixture between inlet/outlet and farfield boundaries, that is usually not a good idea, on your picture I don't see where you are putting the farfield?
CFL 200 is probably too much to start with.
